src/main/java/com/linkedin/replica/serachEngine/models/Post.java

```diff
@@ -4,5 +4,33 @@
  * Post Model
  */
 public class Post {
+	private String id;
+	private String text;
+	private String authorId;
```

Comment:
Why do we have the IDs as strings? didn't we agree that they will be `int` to be compatible with corresponding entries in SQL?